src/components/resource-editor/BlockContentEditorUnsupported.vue

```diff
@@ -29,13 +21,24 @@
   </div>
 </template>
 
-<script lang="js">
-/* eslint-disable @typescript-eslint/explicit-module-boundary-types,@typescript-eslint/strict-boolean-expressions */
-
-import {lang} from '@/lib/filters/lang'
+<script lang="ts">
+import {mixins} from 'vue-class-component'
+import {Component, Prop} from 'vue-property-decorator'
+import Lang from '@/lib/filters/lang'
 
-export default {
-  mixins: [lang],
-  props: ['block', 'blockTypes'],
```

Comment:
- [ ] if we remove `blockTypes`, then let's update any reference in the code base. Remove it in https://github.com/FLOIP/flow-builder/blob/feature/VMO-5609/ts-class-components/src/views/ResourceViewer.vue#L278 as well